examples/des_y1_3x2pt/des_y1_3x2pt_PT.py

```diff
@@ -295,7 +295,8 @@ def run_likelihood() -> None:
     [a.legend(fontsize="small") for a in ax]
 
     fig.suptitle("PT Cls, including IA, galaxy bias, magnification")
-    fig.savefig("plots/pt_cls.png", facecolor="white", dpi=300)
+    with open("pt_cls.png", "w+b") as png_file:
+        fig.savefig(png_file, facecolor="white", dpi=300)
```

Comment:
Why the context manager instead of using the file name directly in `savefig`?

Reply:
Habit. I prefer to make it obvious the resource is “cleaned up” (in this case, the file closed) rather than having a reader need to know whether the routine being called does the right thing.

Reply:
If it's just here that fine but generally I'd argue having a context manager for `savefig` is overkill and likely to confuse users (what happens if you forget the `"b"`?). Might as well throw in a `os.makedirs("plots", exists_ok=True)` to deal with the subdir in that case.